src/helix-ui/elements/index.js

```diff
@@ -0,0 +1,28 @@
+export { HXAccordionElement } from './HXAccordionElement';
```

Comment:
* file moved from `src/helix-ui/elements.js`
* simplified import paths from `./elements/<HXFileName>.js` to `./<HXFileName>`